x-pack/plugins/apm/public/components/app/Main/UpdateBreadcrumbs.tsx

```diff
@@ -27,15 +27,26 @@ function getTitleFromBreadCrumbs(breadcrumbs: Breadcrumb[]) {
 
 class UpdateBreadcrumbsComponent extends React.Component<Props> {
   public updateHeaderBreadcrumbs() {
+    const { basePath } = this.props.core.http;
     const breadcrumbs = this.props.breadcrumbs.map(
       ({ value, match }, index) => {
+        // Remove trailing slash from root route
+        const path = match.url === '/' ? '' : match.url;
```

Comment:
This won't be needed after this is fixed https://github.com/elastic/kibana/issues/75051